googlepay/src/main/java/com/adyen/checkout/googlepay/internal/ui/DefaultGooglePayDelegate.kt

```diff
@@ -106,6 +108,7 @@ internal class DefaultGooglePayDelegate(
             paymentMethod,
             componentParams,
         ) { isAvailable, _ ->
+            this.isAvailable = isAvailable
```

Comment:
What do you think of storing this in the `outputData`, similar to how we store `paymentData`?

Reply:
`outputData` is generally used to communicate with the view, so I'm not sure if it makes sense to store state in there 🤔

Reply:
I get that point. In that case I guess `paymentData` should also be removed from there and it is not part of this task. Let's leave it as it is then.